src/oas.js

```diff
@@ -137,34 +162,39 @@ function filterPathMethods(pathMatches, targetMethod) {
   const regExp = pathToRegexp(targetMethod);
   return pathMatches
     .map(p => {
-      const captures = Object.keys(p.ref).filter(r => regExp.exec(r));
+      const captures = Object.keys(p.operation).filter(r => regExp.exec(r));
 
       if (captures.length) {
         const method = captures[0];
+        p.url.method = method.toUpperCase();
+
         return {
           url: p.url,
-          ref: p.ref[method],
+          operation: p.operation[method],
         };
       }
       return undefined;
     })
     .filter(p => p);
 }
 
-function findTargetPath(pathMatches) {
+function findTargetPath(pathMatches, index) {
   let minCount = Object.keys(pathMatches[0].url.slugs).length;
-  let candidate;
+  let logOperation;
```

Comment:
Let's rename this to `operation` and not have any mention of logs in this package.

```suggestion
  let operation;
```

Reply:
I have mixed feelings about this, because we need to identify a target operation in our FE. Is there another agnostic name that you can suggest?

Reply:
This isn't necessary anymore! It's been removed entirely.